lib/ex_unit/test/ex_unit/doc_test_test.exs

```diff
@@ -307,6 +371,9 @@ defmodule ExUnit.DocTestTest do
   end
 
   test "doctest failures" do
+    # When adding or removing lines in this file above this line,
+    # the line number in the second line of the doctest failure output tests (test/ex_unit/doc_test_test.exs:)
```

Comment:
Oh, looks like GH has partially eaten my comment about this comment. :bowtie:
I suggested to use `test/ex_unit/doc_test_test.exs:<LINE>`.